.lintr

```diff
@@ -2,5 +2,8 @@ linters: lintr::linters_with_tags(tags = NULL, indentation_linter = NULL)
 encoding: "UTF-8"
 exclusions: list(
     # excluded from all lints:
-    "tests/testthat.R"
+    "tests/testthat.R",
+    "vignettes/babelquarto.qmd" = list(
```

Comment:
what happened?

Reply:
I had a `library(babelquarto)` at some point in the _Getting started_ vignette but it's no longer the case. I can remove this again.